src/libopensc/cards.h

```diff
@@ -272,11 +272,15 @@ enum {
 	SC_CARD_TYPE_DTRUST_V4_1_M100,
 	SC_CARD_TYPE_DTRUST_V4_4_STD,
 	SC_CARD_TYPE_DTRUST_V4_4_MULTI,
-	SC_CARD_TYPE_DTRUST_V5_1_STD,
+
+  SC_CARD_TYPE_DTRUST_V5_1_STD,
```

Comment:
probably some artifact of a rebase?
```suggestion
	SC_CARD_TYPE_DTRUST_V5_1_STD,
```

Reply:
[resolved](https://github.com/OpenSC/OpenSC/pull/3423/commits/3079494a13d276cc87e99707e211acd5a2abc3ea#diff-4bdb71298efcb4fa79a57689e24cf9ad08629bacaf83df012a567026cd39ed3eR276)